group_vars/cinder.yml

```diff
@@ -3,3 +3,7 @@
 
 cinder_volume_driver: cinder.volume.drivers.lvm.LVMVolumeDriver
 cinder_backend_name: cinder-volumes
+
+# variables for pre-flight-checks
+cinder_image_conversion_cache: /opt/pf9/pf9-cindervolume-base/lib/python2.7/site-packages/cinder/conversion
+cinder_image_cache_min_free_space: 20000000
```

Comment:
~20 GB? Where did the baseline for the min come from?

Reply:
Just made it up.  It's probably not big enough, but it depends on the size of the customer's images.  Note that in can be over-ridden using "./INSTALL -e "cinder_image_cache_min_free_space=30000000" (for 30GB'ish)